dockerfiles/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM openjdk:8
+ARG HIVE_VERSION=3.0.0.13.3
+ARG HADOOP_VERSION_EE=3.2.0.12-EE-SNAPSHOT
+ARG PROMETHEUS_EXPORTER_VERSION="0.12.0"
+ARG USER="hive"
+ARG USERID="1515"
+ARG GROUP="hive"
+ARG GROUPID="1505"
+ARG HADOOP_GROUP_ID="1234"
+ARG METASTORE_ID="1511"
```

Comment:
that is the certs user id 
https://github.com/logicalclocks/kagent-chef/blob/master/attributes/default.rb#L10C1-L13C60